blocks/article-grid/article-grid.css

```diff
@@ -0,0 +1,126 @@
+/* use styles based on live milo libs */
+@import "https://main--milo--adobecom.hlx.live/libs/blocks/section-metadata/section-metadata.css";
+@import "https://main--milo--adobecom.hlx.live/libs/blocks/media/media.css";
+@import "../banner/banner.css";
+
+.article-grid {
+  position: relative;
+  opacity: 0;
+  padding-bottom: var(--spacing-l);
+}
+
+/* spinner styles in style */
+.article-grid .loading-container {
+  width: 100%;
+  height: 70vh;
+  position: relative;
+}
+
+.article-grid .article-grid-result {
+  opacity: 0;
+  transition: opacity 0.3s ease-in-out;
+}
+
+.article-grid.loading {
+  opacity: 1;
+}
+
+.article-grid.ready .loading-container {
+  display: none;
+}
+
+.article-grid.ready .article-grid-result {
+  opacity: 1;
+}
+
+.article-grid .media .text .description {
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  min-height: 3.2rem;
+}
+
+.article-grid .media {
+  padding: 0;
+}
+
+.article-grid a.media .image {
+  border-radius: 8px;
+}
+
+.article-grid .media .image img {
+  width: 100%;
+  aspect-ratio: 16 / 9;
+  object-fit: cover;
+  border-radius: 8px;
+  overflow: hidden;
+}
+
+.article-grid .con-button::first-letter {
+  text-transform: capitalize;
+}
+
+.article-grid a.media:hover .con-button {
+  background-color: var(--color-black);
+  border-color: var(--color-black);
+  color: var(--color-white);
+  text-decoration: none;
+}
+
+div.article-grid .media .foreground > div.media-row {
+  padding: 0;
+}
+
+.article-grid .banner {
+  max-width: none;
+  aspect-ratio: 1 / 1;
+}
+
+.article-grid .banner,
+.article-grid .banner .banner-contents {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+}
+
+.article-grid .banner .content-wrapper {
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+}
+
+.article-grid .load-more-container {
+  grid-column: 1 / -1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@media (min-width: 700px) {
+  .article-grid .article-grid-result {
+    gap: var(--spacing-l) var(--spacing-m);
+  }
+
+  .article-grid.hide-odd-item-on-tablet .article-grid-result > a:last-of-type:nth-child(odd) {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  .article-grid .media .text h2 {
+    display: -webkit-box;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    min-height: 2.8rem;
+  }
+}
+
+@media (min-width: 1070px) {
```

Comment:
Odd breakpoints that don't follow what's currently in Milo.

Reply:
updated to use only milo breakpoints (`600px`, `900px`, `1200px`), thanks